src/listContext.js

```diff
@@ -41,9 +42,7 @@ const ListContextProvider = props => {
 
   function addItem(name, nextExpectedPurchase) {
     if (!isDuplicate(name)) {
-      let db = firebase.firestore();
-      let itemsRef = db.collection('items');
-      itemsRef.doc().set({ name, token, nextExpectedPurchase });
+      firestore.collection('items').add({ name, token, nextExpectedPurchase });
```

Comment:
Could you use the `itemsRef` variable you created above here? Or maybe you just delete where that var is declared?

Reply:
Done https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/58/commits/3af0609a32bf3796b598372f1b82d7787042b0e2 😃